addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -5,164 +5,163 @@
+                                                class="btn btn-secondary"/>
                                     </div>
-                                    <div class="d-flex gap-1 action_buttons" colspan="3">
-                                        <div class="mt-3"
-                                            invisible="account_peppol_proxy_state != 'not_verified'">
-                                            <button name="button_send_peppol_verification_code"
-                                                    type="object"
-                                                    string="Verify phone number"
-                                                    class="btn btn-primary"/>
-                                        </div>
-                                        <widget name="peppol_settings_buttons"
-                                                invisible="account_peppol_proxy_state not in ('not_registered', 'sent_verification', 'pending', 'manually_approved', 'active')"/>
-                                        <div class="mt-3"
-                                            invisible="account_peppol_proxy_state in ('not_registered', 'active', 'rejected', 'canceled')">
-                                            <button name="button_cancel_peppol_registration"
-                                                    type="object"
-                                                    string="Cancel registration"
-                                                    class="btn btn-secondary"/>
-                                        </div>
-                                        <div class="mt-3"
-                                            invisible="account_peppol_proxy_state != 'canceled' or account_peppol_migration_key">
-                                            <button name="button_reopen_peppol_registration"
-                                                    type="object"
-                                                    string="Retry registration"
-                                                    class="btn btn-secondary"/>
-                                        </div>
+                                    <div class="mt-3"
+                                         invisible="account_peppol_proxy_state != 'canceled' or account_peppol_migration_key">
+                                        <button name="button_reopen_peppol_registration"
+                                                type="object"
+                                                string="Retry registration"
+                                                class="btn btn-secondary"/>
                                     </div>
                                 </div>
                             </div>
                         </div>
                     </div>
                 </div>
-                </block>
-            </block>
+            </div>
+            <field name="module_account_peppol" position="attributes">
```

Comment:
add a little comment somewhere to explain why we need this plz

Reply:
Comment added beneath